versioned_docs/version-1.6/release-notes.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_label: Release Notes
+title: ''
+---
+
+<head>
+  <link rel="canonical" href="https://elemental.docs.rancher.com/release-notes"/>
+</head>
+
+# Release Notes
+
+The Elemental project stack is made of various components such as the `Operator` and `UI` for example.
+
+Each of these components have an independent development lifecycle with its own versioning. Once a new version is ready, meaning it fully integrates with the others components of the Elemental project stack, a release is made.
+
+Here's the different components, their latest version and a link to the respective release notes on GitHub:
+
+| Name                                                                 | Version | Release Notes                                                                |
+|----------------------------------------------------------------------|---------|------------------------------------------------------------------------------|
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)    |
+| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v2.1.1  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v2.1.1)     |
+| [Elemental Linux](https://github.com/rancher/elemental)              | v2.1.1  | [Link](https://github.com/rancher/elemental/releases/tag/v2.1.1)             |
+| [Elemental UI](https://github.com/rancher/elemental-ui)              | v1.3.1  | [Link](https://github.com/rancher/elemental-ui/releases/tag/elemental-1.3.1) |
```

Comment:
This is not existent yet, but makes sense as 1.3.1 should be out soon I guess 👍🏼